classes/i18n/LocaleConversion.php

```diff
@@ -34,51 +35,43 @@ class LocaleConversion
      */
     protected static function getISO6392b(): array
     {
-        $iso6392BFile = Core::getBaseDir() . '/' . PKP_LIB_PATH . '/lib/vendor/sokil/php-isocodes-db-i18n/databases/iso_639-2.json';
-        if (!file_exists($iso6392BFile)) {
-            return [];
+        $key = __METHOD__ . 'iso639-2b';
+        $iso6392bFile = Core::getBaseDir() . '/' . PKP_LIB_PATH . '/lib/vendor/sokil/php-isocodes-db-i18n/databases/iso_639-2.json';
+        if (!file_exists($iso6392bFile)) {
+            $iso6392bFile = Core::getBaseDir() . '/' . PKP_LIB_PATH . '/lib/iso_639-2.json';
         }
-        return json_decode(file_get_contents($iso6392BFile), true);
+        return Cache::rememberForever($key, function () use ($iso6392bFile) {
```

Comment:
I think the key should consider the modified date of the file.

Reply:
If I use the date modified from the iso file as the key, and I store it forever (rememberForever), will the old cache files keep to exist?
Maybe then to use some expiration date, e.g. 1 year?

Reply:
Yep, it's better to add an expiration date.

p.s. At this moment, we don't have a job to clear the expired cache.